.editorconfig

```
@@ -0,0 +1,10 @@
+[*.json]
```

In general, it seems that `root = true` is added to the project root file.
https://github.com/editorconfig/editorconfig/blob/b52217517b2c5b2c3b6a0daa7260983d409e7e6b/.editorconfig#LL2-L2C12

```suggestion
root = true

[*.json]
```